packages/runner-shared/src/iframe/iframe-model.js

```diff
@@ -216,7 +218,28 @@ export class IframeModel {
     this.state.messageType = 'warning'
   }
 
-  _storeOriginalState () {
+  _storeOriginalState (snapshotUrl) {
+    if (!this.isAUTSameOrigin()) {
```

Comment:
This only happens if the AUT ends in a cross origin state that the primary doesn't have access to. In this case, the final snapshot request from the primary is send out to the spec bridges. The spec bridge that matches the origin policy will take a snapshot and send it back to the primary for the runner to store in `originalState`

Reply:
I can and will 😄 . added in 416d3c8854